src/App.css

```diff
@@ -1,4 +1,41 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 .App {
-  display: grid;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+}
+.header {
+  position: absolute;
+  padding: 30px;
+  width: 100%;
+  background-color: #e8abb9;
+  color: #ffffff;
+}
+.main {
   height: 100vh;
```

Comment:
100vh means the main element has a height of 100% of your viewport. as header and footer also take some space on your viewport, you better place 100vh in your App element.

Reply:
noted ✔️